tests/unit/test_workspace_work_item_properties.py

```
@@ -119,3 +132,48 @@ def test_create_update_option_for_workspace_property(
+
+            client.workspace_work_item_properties.options.delete(
+                workspace_slug, prop.id, opt.id
+            )
+            remaining = client.workspace_work_item_properties.options.list(
+                workspace_slug, prop.id
+            )
+            assert all(o.id != opt.id for o in remaining)
+        finally:
+            try:
+                client.workspace_work_item_properties.delete(workspace_slug, prop.id)
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)